Maquetado/scss/details.scss

```
@@ -5,20 +5,32 @@
 }
 
 .book {
-  align-items: center;
   background-color: $white;
   box-shadow: 4px 4px 10px 0 $black-transparent;
   display: flex;
-  flex-flow: column wrap;
+  flex-wrap: wrap;
   height: 100%;
   margin: 0 auto;
-  max-height: 415px;
   max-width: 1000px;
   width: 100%;
 }
 
+.book-img,
+.book-info {
+  height: 100%;
+  margin: 20px;
+}
+
 .book-info {
-  margin: 10px auto;
+  margin-bottom: auto;
+  margin-right: auto;
```

Replace this by
```suggestion
  margin-right: 50px;
```
as in [inVision](https://projects.invisionapp.com/prototype/Wolox-Books-cjw2jveux003x0v01374uqeq8/inspect/447d17df)